pandas/core/categorical.py

```diff
@@ -879,8 +874,22 @@ def rename_categories(self, new_categories, inplace=False):
 
         Parameters
         ----------
-        new_categories : Index-like or dict-like (>=0.21.0)
-           The renamed categories.
+        new_categories : array-like or dict-like
```

Comment:
A bit of a nitpick: new_categories is listed as "array-like" here, but in the description below the term "list-like" is used instead, and then it's switched back to "array-like" further down in the examples section.  Might be nice to make the terminology consistent, but certainly not something that's super critical.

Reply:
Thanks, fixed.